frontend/src/views/PreparationView.tsx

```diff
@@ -0,0 +1,80 @@
+
+import * as React from 'react';
+import { Chip, NavigationStructure, NavigationDrawer } from '@equinor/fusion-components';
+import { Barrier, Answer, Evaluation, Question, } from '../api/models';
+import QuestionAndAnswerForm from '../components/QuestionAndAnswer/QuestionAndAnswerForm';
+import { Box } from '@material-ui/core';
+
+interface BarrierQuestionsProps
+{
+    title: string
+    questions: Question[]
+}
+
+const BarrierQuestions = ({title, questions}: BarrierQuestionsProps) => {
+    return (
+        <>
+            <h2>{title}</h2>
+            {questions.map((question) => {
+                return (
+                    <>
+                        <hr style={{ height: "1", color: "lightgray" }}/>
+                        <QuestionAndAnswerForm
+                            question={question}
+                            answer={question.answers[0]}
+                            questionNumber={1}
+                            onAnswerChange = {(answer: Answer) => {
+
+                            }}
+                        />
+                    </>
+                )
+            })}
+            <br/>
+        </>
+    )
+}
+
+interface PreparationViewProps
+{
+    evaluation: Evaluation
+}
+
+const PreparationView = ({evaluation}: PreparationViewProps) => {
+    const [selectedBarrier, setSelectedBarrier] = React.useState<string>(Object.values(Barrier)[0]);
+    const [structure, setStructure] = React.useState<NavigationStructure[]>(
+        Object.entries(Barrier).map(([shortName, barrierName]) => {
+            return {
+                id: barrierName,
+                type: 'grouping',
+                title: barrierName,
+                icon: <>{shortName}</>,
+                aside: <Chip title="3/10" />
+            }
+        })
+    );
+
+    return (
+        <Box display="flex" flexDirection="row">
+            <Box flexGrow={0}>
+                <NavigationDrawer
+                    id="navigation-drawer-story"
+                    structure={structure}
+                    selectedId={selectedBarrier}
+                    onChangeSelectedId={(selectedItem) => setSelectedBarrier(selectedItem)}
+                    onChangeStructure={(newStructure) => {
+                        setStructure(newStructure);
+                    }}
+                />
+            </Box>
+            <Box>
+                <div style={{margin: 20}}>
```

Comment:
We could do 
```suggestion
            <Box m={20}>
```
here, but how about we just stick to one method and use the MUI `Container`?

Reply:
Hmm, I'm not sure how you reproduce this behaviour with Container?